example/Assets/Standard Assets/Bugsnag/Bugsnag.cs

```diff
@@ -378,10 +384,28 @@ public enum Severity {
 
     void Awake() {
         DontDestroyOnLoad(this);
-        Init(BugsnagApiKey);
+        Init(null);
+    }
+
+    public void Init() {
+		Init(null);
     }
 
-    public void Init(string apiKey)
+	public void Init(string apiKey)	{
+		// Try to work out which API key to use
+		if (!String.IsNullOrEmpty(apiKey)) {
+			InitInternal(apiKey);
+		} else if (!String.IsNullOrEmpty(BugsnagApiKey)) {
+			InitInternal(BugsnagApiKey);
+		} else if (!String.IsNullOrEmpty(BugsnagApiKeyStatic)) {
+			InitInternal(BugsnagApiKeyStatic);
+		} else {
+			// Will cause the native notifiers to throw an exception on Register
+			InitInternal(null);
```

Comment:
Do we want to cause exceptions here?  Specifically on awake, maybe someone might want to initialise at a later stage. Maybe we shouldn't initialise it at all, so its similar to what would happen if you haven't called register in the client notifier. We could record an error log to indicate its not configured correctly.